src/qpython.h

```diff
@@ -131,6 +131,61 @@ class QPython : public QObject {
         evaluate(QString expr);
 
         /**
+         * \brief Asynchronously import objects from Python module
+         *
+         * Imports objects, given by list of names, from Python module asynchronously.
+         * The function will return immediately. If the module is successfully imported,
+         * the supplied \a callback will be called. Only then will the
+         * imported module be available:
+         *
+         * \code
+         * Python {
+         *     Component.onCompleted: {
+         *         importNames('os', ['path'] function() {
```

Comment:
Missing a `,` between `]` and `function`. Also, it returns a boolean that the function could get as parameter:

```
importNames('os', ['path'], function (success) {
    if (success) {
        // You can use the "path" submodule here
    } else {
        console.log('Importing failed')
    }
})
```